services/ui-src/src/forms/mcpar/mcpar.json

```
@@ -149,7 +149,7 @@
             "id": "aap",
             "fields": [
               {
-                "id": "aap-1",
+                "id": "planName",
```

Each report will have lots of different plans added and this field will be pulled into entity pages, so:
```suggestion
                "id": "plans",
```

---

@braxex so what would multiple plans look like